app/controllers/follows_controller.rb

```diff
@@ -79,17 +79,25 @@ def create
     render json: { outcome: @result }
   end
 
-  def update
-    @follow = Follow.find(params[:id])
-    authorize @follow
-    @follow.explicit_points = follow_params[:explicit_points]
-    redirect_to dashboard_following_path if @follow.save
+  def bulk_update
```

Comment:
**suggestion (non-blocking):** This is due to my personal dislike for non-REsTful actions in Rails controllers, but I'd probably do a nested `BulkUpdatesController` with a resourceful route declaration in `routes.rb`, see http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/